modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestWithRefreshToken.siddhi

```diff
@@ -0,0 +1,74 @@
+@App:name("PublishHttpOAuthRequestWithRefreshToken")
+@App:description("Send an oauth events via HTTP transport with refresh token")
```

Comment:
Here also remove 'oauth events' part (we are sending HTTP events to an OAuth-protected endpoint). 

```suggestion
@App:description("Send HTTP events to an OAuth-protected endpoint. Here we use refresh token grant type. ")
```